src/detail-page/components/charts/DevOpsChart.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { FC } from 'react';
+import { Pie } from 'react-chartjs-2';
+import { devOps } from '../../../data/charts';
+import { chartColors } from '../../../data/charts/chartColors';
+import { Chart } from './Chart';
+import { notEmpty } from '../../../shared/utils/helpers';
+
+const { data: sourceData, labelText, source, title: chartTitle, type } = devOps;
+
+const chartLabels = sourceData.map(({ label }) => label);
+const chartValues = sourceData.map(({ value }) => value);
+const chartBackgroundColors = sourceData
+  .map(({ color }) => (color ? chartColors[color].light : null))
+  .filter(notEmpty);
+const chartBorderColors = sourceData
+  .map(({ color }) => (color ? chartColors[color].dark : null))
+  .filter(notEmpty);
+
+const areColorsDefined = !sourceData.some(({ color }) => color === null);
+
+const ChartData: Chart.ChartData = {
+  labels: chartLabels,
+  datasets: [
+    {
+      data: chartValues,
+      borderWidth: 1,
+      ...(labelText && {
+        label: labelText,
+      }),
+      ...(areColorsDefined && {
+        backgroundColor: chartBackgroundColors,
+        borderColor: chartBorderColors,
+      }),
+    },
+  ],
+};
+
+const ChartOptions: Chart.ChartOptions = {
+  legend: {
+    display: true,
+  },
+  responsive: true,
+  maintainAspectRatio: true,
+  tooltips: {
+    callbacks: {
+      label(tooltipItem, data) {
+        const { datasets, labels } = data;
+        const tooltipDatasetIndex = tooltipItem.datasetIndex!;
+        const tooltipIndex = tooltipItem.index!;
+        const tooltipLabel = datasets![tooltipDatasetIndex].label || '';
+        const tooltipValue = datasets![tooltipDatasetIndex].data![tooltipIndex];
+        const valueLabel = labels![tooltipIndex];
+        return ` ${valueLabel}: ${tooltipValue} ${tooltipLabel}`;
+      },
+    },
+  },
+};
+
```

Comment:
I remember that @klaraborowska talked about her experience with tooltips so I'm open to serious refactor this solution if needed!

Reply:
![azure](https://user-images.githubusercontent.com/29304012/117396838-1c509700-aefb-11eb-8df5-b68d7de521df.png)